lisa/tools/modprobe.py

```diff
@@ -80,14 +81,22 @@ def load(
         self,
         modules: Union[str, List[str]],
         dry_run: bool = False,
+        skip_loaded: bool = False,
```

Comment:
this variable name is not clear. The method is `load`, but the variable is `skip_loaded`. Is it similar like `force_reload`?

Reply:
Hm, I had added this to handle a failure when trying to modprobe builtin drivers but maybe this is more of a testing issue on my end. I'm going to close and fix my test setup and we can re-design this in a smarter way if we hit the problem again.